tests/scripts/test_pkgdev_mask.py

```diff
@@ -199,3 +199,32 @@ def test_last_rites(self):
                 cat/pkg
             """)
             self.masks_path.write_text("")  # Reset the contents of package.mask
+
+    def test_mask_bugs(self):
+        removal_date = self.today + timedelta(days=30)
+        today = self.today.strftime('%Y-%m-%d')
+        removal = removal_date.strftime('%Y-%m-%d')
+        for bugs in (['-b', '42'], ['--bug', '42'], ['-b', '42', '-b', '43'], ['--bug', '42', '--bug', '43']):
```

Comment:
I'd use a nested loop here with the outer level looping over the option names and then inner level looping over the args and expected, relevant output.

Reply:
I changed it up, but maybe not in the most/best idiomatic way.